xla/service/gpu/gpu_layout_assignment.cc

```diff
@@ -125,9 +127,10 @@ HeuristicLayoutAssignment(const HloInstruction* instr,
     return kAllNHWC;
   }
 
+  const bool isFloat16 = (input_ty == F16) || (input_ty == BF16);
```

Comment:
CI is complaining about undefined variable if the non TENSORFLOW_USE_ROCM define is active for a CPU build. Please fix.